frontend/renderer/plugins/apollo.ts

```diff
@@ -50,7 +50,7 @@ const cache = new InMemoryCache()
 const errorLink = onError(({ graphQLErrors }) => {
   if (graphQLErrors) {
     graphQLErrors.forEach(({ extensions }) => {
-      if (extensions.code === 'UNAUTHENTICATED') {
+      if (extensions !== null && extensions?.code === 'UNAUTHENTICATED') {
```

Comment:
Is this not the same as
```suggestion
      if (extensions?.code === 'UNAUTHENTICATED') {
```
?

Reply:
Without the check if `extensions`' is undefined, the type related error 

> TS18048: 'extensions' is possibly 'undefined'.

still occurs (as in https://github.com/dreammall-earth/dreammall.earth/actions/runs/10089990430/job/27898496759).
Try it locally with `if (extensions?.code === 'UNAUTHENTICATED') {` and run `npm install && npm run build`